NitroxServer/GameLogic/Entities/EntitySimulator.cs

```diff
@@ -78,7 +89,26 @@ private void AssignEntitiesToNewPlayers(Player oldPlayer, List<Entity> entities,
                     {
                         Log.Info("Player " + player.Name + " has taken over simulating " + entity.Id);
                         ownershipChanges.Add(new SimulatedEntity(entity.Id, player.Id, true, DEFAULT_ENTITY_SIMULATION_LOCKTYPE));
-                        return;
+                        break;
+                    }
+                }
+            }
+        }
+
+        private void AssignEntitiesToNewPlayers(Player oldPlayer, List<NitroxId> entities, List<SimulatedEntity> ownershipChanges)
```

Comment:
Looks like the diff is messed up.  this function is duplicated.

Reply:
Changed the name of the method to show that it is not the same as the other.